tests/wpt/metadata/html/semantics/embedded-content/media-elements/event_timeupdate.html.ini

```diff
@@ -0,0 +1,2 @@
+[event_timeupdate.html]
+  expected: CRASH
```

Comment:
This was an intermittent CRASH.

```
  ▶ OK [expected CRASH] /html/semantics/embedded-content/media-elements/event_timeupdate.html
```

Reply:
Found the original crash in the log from the nightly WPT-update task on https://treeherder.allizom.org/#/jobs?repo=servo-master&selectedTaskRun=XAdWcdjHS56fAqRqHI15uw.0 and filed #27161.

Reply:
/me noted this trick to trace the original crash 👀